addons/website/data/website_data.xml

```diff
@@ -166,17 +166,14 @@
                         <section class="s_text_block pt40 pb40 o_colored_level " data-snippet="s_text_block">
                             <div class="container s_allow_columns">
                                 <div class="row">
-                                    <div class="col-lg-7 col-xl-6 me-lg-auto">
-                                        <span class="d-block fa fa-4x fa-thumbs-up mx-auto rounded-circle bg-primary"/><br/>
-                                        <h1 class="text-center">Thank You!</h1>
-                                        <div class="s_hr pb16 pt16" data-snippet="s_hr" data-name="Separator">
-                                            <hr class="w-50 mx-auto border-dark"/>
+                                    <div class="col-lg-7 col-xl-6 me-lg-auto text-center">
+                                        <div class="d-inline-block mx-auto p-4">
+                                            <i class="fa fa-paper-plane fa-2x mb-3 rounded-circle text-bg-success" role="img" aria-label="Success" title="Success"/>
```

Comment:
Actually, I do think that since this not an a picture but mere text (at least by for screen readers), the `role="img"` attribute is more appropriate.  Also, ARIA guidelines are quite clear about icon fonts, see https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA24.

Reply:
_Disclaimer/Clarification: Your [initial message](https://github.com/odoo/odoo/pull/166138#discussion_r1662394025) has been cited in DMs and other PRs as a "blocking point" for the merge, so I found more appropriate to reply here, even if my comment it's more about the "general approach" and it's not strictly related to the scenario of this PR._ 😉

You make a fair point about the distinction between icons that have meaning Vs those that are purely decorative. I agree that when the content is **not** editable, the icon is just decorative and the text already conveys the meaning (like in the case of this PR), then `role="presentation"` can be the better choice 👍 

However, in the case of editable content (eg. snippets), we don't really know if the user uses an icon to convey a meaning. As a general approach, I think that `role="img"` (alone) it's not perfect but "safer" as it provides more semantic meaning than `role="presentation"`. The rationale is that screen readers will still see the element as an "image", that's more informative than treating it as a purely presentational element (=> ignoring it).
Having an `aria-label` (+ `aria-hidden`+ `.sr-only` text at support..) would be ideal, but it's probably not feasible without making the edition process hyper-complex.

That said, things can get nuanced and I'm not an expert neither 😅🤷🏻‍♂️
My point is more that I don't think this should be a "blocking point" issue that could block a PR 🥲

Let me know if you still prefer `role="presentation"` for editable content.  I'll share the outcome with the team, thanks 🙂

Reply:
> My point is more that I don't think this should be a "blocking point" issue that could block a PR 🥲

Well... depends what you mean by "blocking point". Sure, we can just always merge and fix later but... :wink:  If you mean "point to consider" then:

- `title="XXX"` was a blocking point to me, we don't want polluting titles appearing on hover of random things in pages, after seeing that, my eyes were drawn to `role` and `aria-label`.
- `aria-label="Success"` was a lesser blocking point to me too: we don't want polluting transifex with useless things to translate. Plus, from experience, `aria-label="XXX"` gets copied everywhere without caring (not the case here but it would be another occurrence that people would have copied without caring).
- `role="XXX"`, indeed it's probably not a big blocking point but it should be. Once an user edit the pages, we are mostly not compliant anyway unfortunately (since we don't allow to set up aria-label etc) but we often make PR about fixing those the way google recommends it (improving the accessibility score on google page speed for default Odoo). That said, your "editable or not editable" nuance makes sense. In this case, I would treat it as "not editable" anyway though: I am sure 99.9% of users don't modify their thank you page, and, if they do, the icon they might change still would serve as decoration. For a more popular page... I am not sure. I would still be drawn to always use `role="presentation"` when the original purpose is decoration (EDIT: changed my mind after detailing, I would always use `role="presentation"`):
  - If the user does not change it (probably the most common usecase) => good
  - If the user does change it, I would say it's more likely to keep the same purpose => good
  - If the user does change it, there is a chance that they would remove the original icon and re-create it anyway => bof (it probably won't have any set `role` by the editor in that case?)
  - If the user does change it to have a meaningful purpose, `role="img"` on its own is not that good anyway. It needs an `aria-label` (which the user cannot set using the builder), unless there is text around... in which case, the icon can most likely be considered a decoration anyway.
  - If the user does change the icon and you set up an aria-label previously, your non-editable aria-label might become just wrong.
  - In general: I'd say it should be rare that an icon without `title` has meaning. We are better off hiding them all for screen-readers and miss some than showing them all and making noise but I may be wrong on that one.
  
... well, typing this convinced me even more that, given the state of the builder, I don't think that setting up `role="img" aria-label="XXX"` is ever right in fact (unless non editable and sure it is not a decoration).